compiler/tests/success/arm-m4/intrinsic_tst.jazz

```diff
@@ -22,9 +26,7 @@ fn tst(reg u32 arg0, reg u32 arg1) -> reg u32 {
     n, z, c = #TSTcc(arg0, arg1, !z && n == v, n, z, c); // GT
     n, z, c = #TSTcc(arg0, arg1, z || n != v, n, z, c);  // LE
 
-    arg0 = 0 if n;
-    arg0 = 0 if z;
-    arg0 = 0 if c;
+    n, z, c = #TSTcc(arg0, 3, z, n, z, c);
 
     return arg0;
```

Comment:
Caveat: most of this program is dead code!

Reply:
It does sound more robust to have both, I updated them.